.github/workflows/build-tmp.yaml

```
@@ -0,0 +1,41 @@
+    permissions:
+      packages: write
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: 'Login to GitHub Container Registry'
+        uses: docker/login-action@v1
```

actionlint (1.7.7)

the runner of "docker/login-action@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)